pocs/camera/camera.py

```diff
@@ -242,9 +241,8 @@ def process_exposure(self, info, observation_event, exposure_event=None):
 
         try:
             self.logger.debug("Extracting pretty image")
-            img_utils.make_pretty_image(file_path,
-                                        title=info['field_name'],
-                                        primary=info['is_primary'])
+            img_utils.make_pretty_image(
```

Comment:
These kind of changes are the only ones I'm not super keen on, but I think that's a throw-back to my Perl days. I like to have the params lined up under the first param if possible. Not a big deal though.